pkg/virt-handler/virtwrap/errors.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of the kubevirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2017 Red Hat, Inc.
+ *
+ */
+
+package virtwrap
+
+import (
+	"reflect"
+
+	"github.com/libvirt/libvirt-go"
+)
+
+func checkError(err interface{}, expectedError libvirt.Error) bool {
+	switch err.(type) {
+	case libvirt.Error:
+		return err.Code == expectedError
+	case error:
+		return err.(libvirt.Error).Code == expectedError
+	default:
+		panic(err)
```

Comment:
I would follow the k8s errors.go example, and never panic. If it is not a libvirt error, or not the expected libvirt error, just return false.